content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -102,14 +102,30 @@ Pods, Controllers and Services are critical elements to managing Windows workloa
 
 Docker EE-basic 18.09 is required on Windows Server 2019 / 1809 nodes for Kubernetes. This works with the dockershim code included in the kubelet. Additional runtimes such as CRI-ContainerD may be supported in later Kubernetes versions.
 
-#### Storage
+#### Persistent Storage
 
-Kubernetes Volumes enable complex applications with data persistence and Pod volume sharing requirements to be deployed on Kubernetes. Kubernetes on Windows supports the following types of [volumes](/docs/concepts/storage/volumes/):
+Kubernetes [volumes](/docs/concepts/storage/volumes/) enable complex applications with data persistence and Pod volume sharing requirements to be deployed on Kubernetes. Management of persistent volumes associated with a specific storage back-end or protocol includes actions such as: provisioning/de-provisioning/resizing of volumes, attaching/detaching a volume to/from a Kubernetes node and mounting/dismounting a volume to/from individual containers in a pod that needs to persist data. The code implementing these volume management actions for a specific storage back-end or protocol is shipped in the form of a Kubernetes volume [plugin](/docs/concepts/storage/volumes/#types-of-volumes). The following broad classes of Kubernetes volume plugins are supported on Windows:
```

Comment:
nit: If I read correctly, line 107 may read better with a comma after applications:
... enable complex applications, with data persistence and Pod volume sharing requirements, to be ...